pkgs/by-name/mi/mistral-rs/package.nix

```diff
@@ -111,17 +104,10 @@ rustPlatform.buildRustPackage rec {
       cudaPackages.libcublas
       cudaPackages.libcurand
     ]
-    ++ lib.optionals mklSupport [ mkl ]
-    ++ lib.optionals stdenv.hostPlatform.isDarwin darwinBuildInputs;
+    ++ lib.optionals mklSupport [ mkl ];
 
   cargoBuildFlags =
-    [
-      # This disables the plotly crate which fails to build because of the kaleido feature requiring
-      # network access at build-time.
-      # See https://github.com/NixOS/nixpkgs/pull/323788#issuecomment-2206085825
-      "--no-default-features"
-    ]
-    ++ lib.optionals cudaSupport [ "--features=cuda" ]
+    lib.optionals cudaSupport [ "--features=cuda" ]
```

Comment:
Doesn't `buildRustPackage` accept `buildFeatures` as an argument? That should be more composable